packages/lesswrong/lib/collections/posts/views.ts

```diff
@@ -840,26 +839,27 @@ ensureIndex(Posts,
 );
 
 Posts.addView("nearbyEvents", (terms: PostsViewTerms) => {
-  const yesterday = moment().subtract(1, 'days').toDate();
   const onlineEvent = terms.onlineEvent === false ? false : viewFieldAllowAny
   let query: any = {
     selector: {
       location: {$exists: true},
       groupId: null,
       isEvent: true,
       onlineEvent: onlineEvent,
-      $or: [{startTime: {$exists: false}}, {startTime: {$gt: yesterday}}],
+      $or: [{startTime: {$gt: new Date()}}, {endTime: {$gt: new Date()}}],
       mongoLocation: {
         $near: {
           $geometry: {
                type: "Point" ,
                coordinates: [ terms.lng, terms.lat ]
           },
+          $maxDistance: 80000 // only show in-person events within 50 miles
```

Comment:
This one's nonobvious. If I lived 81km/51mi from a major city, I probably wouldn't want this?

Reply:
50mi seems far for a small meetup but I expect people to go further for bigger ones. I might even do 300mi.

Reply:
Fair enough!

Reply:
Are they really that close? Wow.